logs_sdk/lib/opentelemetry/sdk/logs/export.rb

```diff
@@ -7,6 +7,9 @@
 module OpenTelemetry
   module SDK
     module Logs
+      ExportError = Class.new(OpenTelemetry::Error)
```

Comment:
In order for this to match the convention in traces, this should be a `Logs::Export::ExportError` (needs to be nested under the Export module).

see: https://github.com/open-telemetry/opentelemetry-ruby/blob/92748f2b74c510d55c50b226380f9f1c51db4a4a/sdk/lib/opentelemetry/sdk/trace/export.rb#L9-L14

Reply:
Thank you! Fixed in 425ab1c6ebf4287f43f26a87caaff20c6f603f02